### PR TITLE
[BULK] Add basic IT for bulk

### PR DESCRIPTION
**Pull request by Deep1998**

### CODE REVIEW

**Comment by manitgupta** [comment]
Can we document the Bulk migration CUJs in parallel as well so that we are clear on what CUJs we plan to cover via ITs similar to live and reverse replication - 

**Live**
https://g3doc.corp.google.com/third_party/harbourbridge/g3doc/testing/integration-testing/forward-migration-it.md?cl=head

**Reverse**
https://critique.corp.google.com/cl/636253544

File: cicd/internal/flags/common-flags.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ const (
 // Avoid making these vars public.
 var (
 	modulesToBuild string
-	moduleMap      = map[string]string{ALL: "", SPANNER: "v2/datastream-to-spanner/,v2/spanner-change-streams-to-sharded-file-sink/,v2/gcs-to-sourcedb/,v2/sourcedb-to-spanner/,plugins/templates-maven-plugin"}
+	moduleMap      = map[string]string{ALL: "", SPANNER: "v2/sourcedb-to-spanner/"}
```
[the remark this answers is not in the document]
Comment (Deep1998) [reply cicd/internal/flags/common-flags.go:32]: Nopes will remove, this wws done to run ITs quickly.

**Comment by Deep1998** [comment]
> Can we document the Bulk migration CUJs in parallel as well so that we are clear on what CUJs we plan to cover via ITs similar to live and reverse replication -
> 
> **Live** https://g3doc.corp.google.com/third_party/harbourbridge/g3doc/testing/integration-testing/forward-migration-it.md?cl=head
> 
> **Reverse** https://critique.corp.google.com/cl/636253544

Yes they are currently documented in this [doc](https://docs.google.com/document/d/1LHnwQxbMaQ98R9sSXsiq82GG-fxvc_HeGey4Glzm3RE/edit?tab=t.0#heading=h.is7zcoaw28wx). Will raise a cl for a g3doc once finalised.

**Review by bharadwaj-aditya — APPROVED** [review]
LGTM

**Review by VardhanThigle — APPROVED** [review]

**Pull request merged**